src/main/java/com/google/api/codegen/config/DiscoveryField.java

```diff
@@ -74,8 +78,8 @@ private DiscoveryField(Schema refSchema, DiscoApiModel apiModel) {
     this.apiModel = apiModel;
 
     String simpleName = DiscoGapicParser.stringToName(refSchema.getIdentifier()).toLowerCamel();
-    String namespace = apiModel.getDefaultPackageName();
-    if (isTopLevelSchema(schema)) {
+    this.namespace = apiModel == null ? DEFAULT_NAMESPACE : apiModel.getDefaultPackageName();
```

Comment:
I think we should document when the apiModel might be null.

Reply:
Done.